Assets/BoundingsTimeout2D.cs

```diff
@@ -8,7 +8,6 @@ public class BoundingsTimeout2D : MonoBehaviour
 
     void Start()
     {
-        timeout = 3;
```

Comment:
timeout with a default value of 0 in the editor right? I may just change this to timeout = 0 to be consistent